static/app/components/replays/replayContext.tsx

```diff
@@ -114,11 +114,21 @@ interface ReplayPlayerContextProps extends HighlightCallbacks {
    */
   setSpeed: (speed: number) => void;
 
+  /**
+   * Set the timeline to the specific size
+   */
+  setTimelineSize: (size: number) => void;
```

Comment:
Nit: The comment below does this a bit, but can we update the names even more to indicate that the value is a percentage?

```suggestion
  /**
   * Set the timeline width to the specific scale, starting at 100% and growing larger
   */
  setTimelineScale: (percent: number) => void;
```